agenda/domain/src/main/java/com/nibalk/tasky/agenda/domain/AgendaItem.kt

```diff
@@ -0,0 +1,65 @@
+package com.nibalk.tasky.agenda.domain
+
+import java.time.LocalDateTime
+
+sealed class AgendaItem(
+    val id: String,
+    val title: String,
+    val description: String,
+    val time: LocalDateTime,
+    val remindAt: LocalDateTime,
+    val isDone: Boolean,
+) {
+    data class Event(
+        val eventId: String,
+        val eventTitle: String,
+        val eventDescription: String,
+        val eventFromDateTime: LocalDateTime,
+        val eventToDateTime: LocalDateTime,
+        val eventRemindAt: LocalDateTime,
+        val eventIsDone: Boolean,
+        val eventAttendees: List<EventAttendee>,
+        val eventHostId: String,
+        val eventIsHost: Boolean,
+        val eventPhotos: List<EventPhoto>
```

Comment:
To avoid respecifying all these fields, you can make the fields of the parent class `open` (so they can be overridden) and then override these in the event's constructor.